### PR TITLE
Add pull request template

### PR DESCRIPTION
**Pull request by zeshanarifios**

### CODE REVIEW

File: docs/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+### Description
```
Comment (zahid-arbisoft) [inline docs/PULL_REQUEST_TEMPLATE]: I think file name should be renamed to `pull_request_template.md`, by making template Markdown, in future, we can use features like Markdown formatting, @-mentions, and task lists

File: docs/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+[LP-](https://philanthropyu.atlassian.net/browse/LP-)
```
Comment (zahid-arbisoft) [inline docs/PULL_REQUEST_TEMPLATE]: What if PR handles more than two stories

Comment (zeshanarifios) [reply docs/PULL_REQUEST_TEMPLATE]: It just fills the box with this template when you create PR, So if PR contains 2 stories you can copy paste this link..

**Comment by zeshanarifios** [comment]
@zahid-arbisoft Please review it again, file has been renamed

**Pull request merged**